src/components/CompareResults/RevisionRow.tsx

```diff
@@ -364,6 +365,9 @@ function RevisionRow(props: RevisionRowProps) {
             <div className='platform-container'>
               {platformIcon}
               <span>{platformShortName}</span>
+              {platformShortName === androidPlatform && (
+                <span>&nbsp;{getPlatformVersion(platform)}</span>
+              )}
```

Comment:
make a `getPlatformAndVersion` that calls `getPlatformShortName` and return the full string

use `\u00a0` inside the string as the non-breaking space.